### DISCUSSION TITLE
How to convert a latex file to word via pandoc [withBinaryFile: does not exist (No such file or directory)]

### DISCUSSION THREAD

**Discussion by jyoti-sn** [jgm/pandoc#10496] [Q&A]
Hello, I always get this message [[withBinaryFile: does not exist (No such file or directory)] when I try to convert tex to a docx file. Can someone please tell me what is it that I am doing wrong? 

I've tried the following commands:

pandoc -s Users/nameredacted/Desktop/untitled folder/latexfile.tex -o  docfile.docx

pandoc -s /Users/nameredacted/Desktop/untitled folder/latexfile.tex -o  docfile.docx

pandoc Path:/Users/nameredacted/Desktop/untitled folder/latexfile.tex -o docfile.docx

pandoc -s latexfile.tex -o docfile.docx

**Comment by TomBener** [comment]
You could try this:

```
pandoc -s /Users/nameredacted/Desktop/untitled\ folder/latexfile.tex -o docfile.docx
```

**Comment by jyoti-sn** [comment]
Yes, it worked!! You know idea I've spent the whole day trying to figure this. Just one more question. Can you tell what's wrong in my command with bibliography?
pandoc --citeproc /Users/nameredacted/Desktop/Market_and_Autonomy\ Final_Paper/autonomy_final_paper.tex --bibliography=autonomy_final_paper_bib.bib -o autonomy_converted.docx

I am getting this error message: File autonomy_final_paper_bib.bib not found in resource path

**Reply by TomBener** [reply]
Based on the error message, it seems the bibliography file (`autonomy_final_paper_bib.bib`) can't be found in the current working directory or the resource path. 

There are a few potential fixes:

1. If your .bib file is in the same folder as your .tex file, you need to provide the full path to the bibliography file:

```bash
pandoc --citeproc /Users/nameredacted/Desktop/Market_and_Autonomy\ Final_Paper/autonomy_final_paper.tex --bibliography=/Users/nameredacted/Desktop/Market_and_Autonomy\ Final_Paper/autonomy_final_paper_bib.bib -o autonomy_converted.docx
```

2. Or you could first change your working directory to where your files are:

```bash
cd /Users/nameredacted/Desktop/Market_and_Autonomy\ Final_Paper/
pandoc --citeproc autonomy_final_paper.tex --bibliography=autonomy_final_paper_bib.bib -o autonomy_converted.docx
```

Which folder is your .bib file located in? This will help me provide the most accurate solution.​​​​​​​​​​​​​​​​

**Reply by jyoti-sn** [reply]
Yes, the first method worked. I did have the bib file in the same folder. Thank you so much -- you're a Godsend!
